src/pages/components/typography.mdx

```diff
@@ -16,6 +17,17 @@ components:
 
 ## How to use it
 
+### Block quotation
+
+Use Blockquote to represent a body of text that is from another source.
```

Comment:
Another use case is to bring special emphasis to content found within the body. That's editorial, and we may never see anything like that in product… should it be called out anyways, or would it just cause too much confusion? @steue @vsrixyz

Reply:
Merging per offline convo with @steue. Happy to adjust copy under a separate PR.